ash/src/vk/extensions.rs

```diff
@@ -1487,8 +1487,6 @@ impl KhrSamplerMirrorClampToEdgeFn {
 impl SamplerAddressMode {
     #[doc = "Note that this defines what was previously a core enum, and so uses the 'value' attribute rather than 'offset', and does not have a suffix. This is a special case, and should not be repeated"]
     pub const MIRROR_CLAMP_TO_EDGE: Self = Self(4);
-    #[deprecated = "Alias introduced for consistency with extension suffixing rules"]
-    pub const MIRROR_CLAMP_TO_EDGE_KHR: Self = Self::MIRROR_CLAMP_TO_EDGE;
```

Comment:
Looks like we forgot about this nonstandard deprecation message when yeeting out all other `deprecated` alias in the previous breaking Ash release. Merging this to `master` is fine, but I'll apply the following workaround when backporting this upgrade to `0.37`.

https://github.com/ash-rs/ash/commits/for-0.37-keep-deprecated-alias